frontend/lib/src/components/elements/Skeleton/AppSkeleton.tsx

```diff
@@ -50,7 +56,7 @@ const RawAppSkeleton: FC = () => {
         <TextLineSkeleton width="96%" />
         <TextLineSkeleton width="65%" />
       </ParagraphSkeleton>
-      <SquareSkeleton width="75%" height="9rem" />
+      <SquareSkeleton width="75%" height={element.height + "pt"} />
```

Comment:
Hm, will this change the way that skeletons currently look when rendered as loading elements?

Since Skeletons today are only ever constructed purely on the frontend using the `makeSkeletonElement` helper function, `element.height` will either be `0` or `undefined` here rather than `9rem`

Reply:
I think you might be right, depending on what we do for your other bit of feedback about the default value in python, we will likely need to add some kind of of other default mechanism here as well, or modify `makeSkeletonElement`.